test/unit/updateAction.js

```diff
@@ -300,6 +299,13 @@ var expectedChanges = {
     }
 };
 
+var queryOptions = { type: 'AmbientLightSensor' };
+
+var expectedChanges2 = {
+    actionType: 'APPEND',
```

Comment:
Right syntax in NGSIv2 is camelCase for action types, i.e. "append".

Reply:
fixed in d93078c1a87dfd9e54e00f3021a06c9893620b11